File: fontes/pascal/micro/micro08.pas
program micro08;
 var
 numero: integer;
 begin
 numero := 1;
 while numero <> 0 do
 begin
 write('Digite um numero: ');
 readln(numero);
 if (numero > 10) then
 begin
 writeln('O numero',numero,' eh maior que 10');
 end
 else
 begin
 writeln('O numero ', numero,' eh menor que 10');
 end;
 end;
 end.

File: fontes/c/micro/micro08.c
#include<stdio.h>

int main()
{
 int numero;
 numero = 1;
 do
 {
	 printf("Digite um número: ");
	 scanf("%d", &numero);
	 if(numero > 10)
	 {
	 	printf("O numero %d é maior que 10 \n", numero);
	 }
	 else
	 {
	 	printf("O numero %d é menor que 10 \n", numero);
	 }
 }
 while(numero != 0);

 return 0;
}